src/pages/home/Home.page.jsx

```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { RiHandHeartFill } from "react-icons/ri";
+import { FaShippingFast } from "react-icons/fa";
+import CategoryCard from "../../components/categoryCard/CategoryCard.component";
+import ProductCard from "../../components/productCard/ProductCard.component";
+import feauturedProducts from "../../utils/mocks/featured-products.json";
+import feauturedCategories from "../../utils/mocks/product-categories.json";
+import {
+  StyledDescription,
+  StyledIcon,
+  StyledTitle,
+  StyledTitleInformaton,
+  StyledWrapper,
+  StyledWrapperCategories,
+  StyledWrapperDescription,
+  StyledWrapperInformation,
+  StyledWrapperTitle,
+} from "./Home.styles";
+import Slider from "../../components/slider/Slider.component";
+
+// eslint-disable-next-line react/prop-types
+function HomePage() {
+  const products = feauturedProducts.results;
```

Comment:
Let's always keep an eye for typos when naming our variables. _featuredProducts_